.pre-commit-config.yaml

```diff
@@ -97,18 +97,39 @@ repos:
           - types-tqdm
           - types-flask
 
+  # Check for spelling
+  - repo: https://github.com/crate-ci/typos
+    rev: v1.23.6
+    hooks:
+      - id: typos
+        exclude: .html
```

Comment:
I am not 100% sure this is the best way to go about this. This would hide all potential typos in HTML pages.
I think I get why you put this here because I guess it autocorrects some html elements, but I think the configuration options of the `typos` project are flexible enough to exclude these separately.

Reply:
I now only exclude the (German) `legal.html`.